lianad/src/payjoin/sender.rs

```diff
@@ -123,7 +123,10 @@ pub(crate) fn payjoin_sender_check(db: &sync::Arc<sync::Mutex<dyn DatabaseInterf
             Ok(None) => {
                 log::info!("[Payjoin] Proposal not received yet...");
             }
-            Err(e) => log::warn!("payjoin_sender_check(): {}", e),
+            Err(e) => {
+                log::warn!("payjoin_sender_check(): {}", e);
+                continue;
```

Comment:
Looks like we need something similar to the receiver where the whole body of the for loop is moved to a seperate method. For example `replay_event_log` could fail and break out the loop early